packages/grpc-js-core/src/call.ts

```diff
@@ -38,6 +38,17 @@ export interface Call extends EventEmitter {
       event: 'metadata', listener: (metadata: Metadata) => void): this;
   removeListener(event: 'metadata', listener: (metadata: Metadata) => void):
       this;
+
+  addListener(event: 'status', listener: (status: StatusObject) => void): this;
```

Comment:
The changes to this file do not match the existing implementation, which only emits the status event for calls with response streaming.

Reply:
It seems like that's not the case -- this test depends on `'status'` being emitted for a unary call: https://github.com/grpc/grpc-node/blob/ac186ec8d13a7020b8b81db4c9d413aba7266781/test/api/surface_test.js#L896-L904